internal/service/cognitoidp/user_pool_test.go

```diff
@@ -1744,11 +1775,26 @@ resource "aws_cognito_user_pool" "test" {
   sms_configuration {
     external_id    = %[2]q
     sns_caller_arn = aws_iam_role.test.arn
+    sns_region     = "us-east-1"
   }
 }
 `, rName, externalID)
 }
 
+func testAccUserPoolConfig_smsConfigurationSnsRegion(rName string, snsRegion string) string {
+	return testAccUserPoolSMSConfigurationBaseConfig(rName, snsRegion) + fmt.Sprintf(`
```

Comment:
```suggestion
	return testAccUserPoolSMSConfigurationBaseConfig(rName, "test") + fmt.Sprintf(`
```
the ...`BaseConfig()` function takes a resource name and `externalID` parameter, not the `snsRegion`. That param has to match line 1790 for this test to pass.